src/routes/capacity.js

```diff
@@ -0,0 +1,62 @@
+const express = require('express');
+const { BigNumber } = require('ethers');
+const router = express.Router();
+const { calculateTranche, calculateCapacities } = require('../lib/helpers');
+const { MIN_COVER_PERIOD } = require('../lib/constants');
+
+router.get('/capacity', (req, res) => {
+  const { stakingPools } = req.store.getState();
+
+  const currentTranche = calculateTranche();
+  const startingTranche = calculateTranche(MIN_COVER_PERIOD);
+
+  const result = Object.entries(stakingPools).reduce((acc, [productId, productData]) => {
+    acc[productId] = {};
+    acc[productId].totalCapacity = BigNumber.from(0);
+    acc[productId].initialCapacityUsed = BigNumber.from(0);
+    acc[productId].poolCapacities = {};
+    for (const pool of Object.entries(productData)) {
+      const [poolId, poolData] = pool;
+      const { initialCapacityUsed, totalCapacity } = calculateCapacities(
+        poolData.trancheCapacities,
+        poolData.allocations,
+        startingTranche - currentTranche,
+      );
+      acc[productId].totalCapacity = acc[productId].totalCapacity.add(totalCapacity);
+      acc[productId].initialCapacityUsed = acc[productId].initialCapacityUsed.add(initialCapacityUsed);
+      acc[productId].poolCapacities[poolId] = { totalCapacity, initialCapacityUsed };
+    }
+    return acc;
+  }, {});
```

Comment:
Create `store/selectors.js`. Move this and the contents of the function below to separate functions `selectCapacities` and `selectProductCapacity`. Both functions accept the store as a first argument.